pkg/util/runtime/runtime_linux.go

```diff
@@ -29,7 +31,20 @@ func parseKernelVersionStr(kernelVersionStr string) (semver.Version, error) {
 	// Patch: 0
 	// Pre: [72-generic]
 	// Build: []
-	return semver.Parse(kernelVersionStr)
+	verStrs := strings.Split(kernelVersionStr, ".")
+	switch {
+	case len(verStrs) < 2:
+		return semver.Version{}, fmt.Errorf("unable to get kernel version from %q", kernelVersionStr)
+	case len(verStrs) < 3:
+		verStrs = append(verStrs, "0")
+	}
+	patch := regexp.MustCompilePOSIX(`^[0-9]+`).FindString(verStrs[2])
+	if patch == "" {
+		verStrs[2] = "0"
```

Comment:
Is there a case the kernel release doesn't have patch number?

Reply:
Maybe there is no need to check kernel patch number, the kernel patch usually be numbers, and it will lose prerelease value,  I will remove it.

Reply:
> Is there a case the kernel release doesn't have patch number?

Maybe there is no need to check kernel patch number, the kernel patch usually be numbers, and it will lose prerelease value, I will remove it.